src/util/Whamm.v3

```diff
@@ -62,10 +59,9 @@ component Whamm {
 
 					var pred: WhammPredicate;
 					r.skipWhitespace();
-					if (r.char == '/') {
-						r.req1('/');
+					if (r.opt1('/') != -1) {
 						r.skipWhitespace();
-						pred = WhammPredicate.WhammParam(parseParam0(r));
+						pred = WhammPredicate.new(WhammParam.Call.!(parseParam0(r)));
```

Comment:
It looks like this will fail if the predicate is not a call. How about having a dedicated method to parse calls that enforces the syntax?